src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,11 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        for (User user:users) {
```

Comment:
```suggestion
        for (User user : users) {
```
make sure to format your file (Ctrl Alt L)